pystac/extensions/file.py

```diff
@@ -6,7 +6,10 @@
 from enum import Enum
 from typing import Any, Dict, List, Optional
 
-import pystac
+from pystac.asset import Asset
+from pystac.stac_object import STACObjectType
+from pystac.errors import ExtensionTypeError
+from pystac.item import Item
 from pystac.extensions.base import ExtensionManagementMixin, PropertiesExtension
```

Comment:
Here I would suggest: `from pystac.extentions import base`.  Then it's `base.PropertiesExtension` in the code.